mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loanaccount/LoanAccountFragment.java

```diff
@@ -718,7 +718,7 @@ private void showHideRepaidMonthSpinners(int visibility) {
 
     private void showDefaultValues() {
         interestRatePerPeriod = mLoanTemplate.getInterestRatePerPeriod();
-        loanTermFrequencyType = mLoanTemplate.getInterestRateFrequencyType().getId();
+        loanTermFrequency = mLoanTemplate.getInterestRateFrequencyType().getId();
```

Comment:
@tarun0 default value should be termPeriodFrequencyType id not interestRateFrequencyType id.